data/hai/hai jobs.txt

```diff
@@ -407,7 +407,7 @@ mission "Delivery to Human Space [0]"
 	job
 	repeat
 	description "This human wishes for <cargo> to be delivered to <destination> in human space. Payment is <payment>."
-	cargo random 5 10
+	cargo random 2 2 .29
```

Comment:
The reason these Hai jobs have more fixed cargo requirements is because we're forced into using fixed payments as a result of destinations being across the wormhole. Until #6846 is merged, I'd rather keep the Hai job cargo values more fixed.

Reply:
It's in.
These missions should now allow the use of wormholes in their distance calculation settings, and have the payments use a payment multiplier instead of being a set payment.